google-cloud-firestore/pom.xml

```diff
@@ -63,6 +75,40 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-auth</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-api</artifactId>
+      <version>0.10.1</version>
```

Comment:
We should use a single opencensus version from google-cloud-bom.

Reply:
Yeah I just added it there while integrating tracing into spanner. You will not be able to use 0.10.1 since it depends on grpc-context 1.8.0 while we depend on grpc-context 1.7.0.

Reply:
Ah, I see Cloud Spanner just added one. I will incorporate that change in.

Reply:
Ah I see you excluded grpc-context. Ok you might need to do the same in google-cloud-spanner/pom.xml if you increasing the version of opencensus in google-cloud-bom/pom.xml